index.js

```diff
@@ -13,3 +13,195 @@ xMen.addEventListener('click', () => {
+    live: '#',
+    source: '#',
+    skills: {
+      skill1: 'html',
+      skill2: 'css',
+      skill3: 'javaScript',
+    },
+  },
+
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    title: 'Canopy',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    mobileImg: './assets/Snapshoot Portfolio 2.png',
+    desktopImg: './assets/second-desktop-card.png',
+    live: '#',
+    source: '#',
+    skills: {
+      skill1: 'html',
+      skill2: 'css',
+      skill3: 'javaScript',
+    },
+  },
+
+  {
+    id: 3,
+    name: 'Tonic',
+    title: 'Canopy',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    mobileImg: './assets/Snapshoot Portfolio 3.png',
+    desktopImg: './assets/second-desktop-card.png',
+    live: '#',
+    source: '#',
+    skills: {
+      skill1: 'html',
+      skill2: 'css',
+      skill3: 'javaScript',
+    },
+  },
+
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    title: 'Canopy',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: 'Experimental content creation feature that allows users to add to an existing stay over the course of a day without spamming their friends.',
+    mobileImg: './assets/Snapshoot Portfolio 4.png',
+    desktopImg: './assets/fourth-desktop-card.png',
+    live: '#',
+    source: '#',
+    skills: {
+      skill1: 'html',
+      skill2: 'css',
+      skill3: 'javaScript',
+    },
+  },
+
+];
+
+projectDetails.forEach((projects, index) => {
+  const section = document.createElement('div');
+
+  const isSecond = index === 1;
+  const isLast = index === 3;
+  const reverseClass = isSecond ? 'inner-second-containerr' : '';
+  const marginB = isLast ? 'inner-second-containerl' : '';
+
+  section.innerHTML = `
+    <section class = "second-container">
+      <div class = "inner-second-container ${reverseClass} ${marginB}">
+        <div class = "top-image same ${projects.id}">
+          <img src=" ${projects.mobileImg} " class="mobile-first-card ${projects.id}"></img>
+          <img src="${projects.desktopImg}" class="desktop-first-card ${projects.id}"></img>
+        </div>
+
+        <div class = "text-inner-container same">
+          <h2> ${projects.name}</h2>
+          <div class = "canopy-inner-container">
+            <p id ="text1"> ${projects.title} </p>
+            <ul> 
+              <li> ${projects.stack} </li> 
+              <li> ${projects.year} </li> 
+            </ul>
+          </div>
+
+          <div class = "canopy-daily"> 
+            <p> ${projects.description} </p>
+          </div>
+              
+          <ul class = "canopy-list"> 
+            <li> ${projects.skills.skill1} </li>
+            <li> ${projects.skills.skill2} </li>
+            <li> ${projects.skills.skill3} </li> 
+          </ul>
```

Comment:
- [ ] `required`
- [ ] If I click on the `see project` the pop-up is not responding, please fix it
- [ ] please see `gif screenshot` below 

`click on the GiF to open it`
![Animation2](https://user-images.githubusercontent.com/66230306/233509238-b60b9847-4c1e-4118-be8e-b12e6600765e.gif)